frontend/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mhart/alpine-node:11 AS builder
```

Comment:
Why not use the official node image like this
FROM node:13.8.0
WORKDIR /app
ADD package.json /app
RUN yarn install
...
etc